test/main/php/com-realexpayments-hpp-sdk/utils/ValidationUtilsTest.php

```diff
@@ -2197,9 +2176,9 @@ public function testValidationPassedSuccess() {
 
 		try {
 			ValidationUtils::validate( $hppRequestConverted );
+            $this->assertTrue(true);
```

Comment:
There are some asserts like this. This is needed since in PHPUnit 6, having no asserts on a test throws a warning.